config/affirmation-request-watcher.config.js

```diff
@@ -1,11 +1,25 @@
 require('dotenv').config()
 const baseConfig = require('./base.config')
 const erc20Abi = require('../abis/ERC20.abi')
+const { ERC_TYPES } = require('../src/utils/constants')
+
+const initialChecksJson = process.argv[3]
```

Comment:
could we check if `argv[3]` exists in order to log proper message if `affirmation-request-watcher` was run without this parameter?

Reply:
Yes I think it makes sense. Added a check here https://github.com/poanetwork/token-bridge/pull/145/commits/ff56ec992942160009fdc74367dc0e3b178a6c59